suite-common/test-utils/src/mocks.ts

```diff
@@ -196,7 +201,10 @@ const getConnectDevice = (dev?: Partial<Device>, feat?: Partial<Features>): Devi
  * @param {Partial<Features>} [feat]
  * @returns {TrezorDevice}
  */
-const getSuiteDevice = (dev?: Partial<TrezorDevice>, feat?: Partial<Features>): TrezorDevice => {
+const getSuiteDevice = (
+    dev?: Partial<Omit<TrezorDevice, 'path'> & { path: string }>,
```

Comment:
accepting path as string in mock so that we don't need to initiate branded type in fixtures